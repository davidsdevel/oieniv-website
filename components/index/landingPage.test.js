/* eslint-env jest */
import React from "react";
import LandingPage from "./landingPage";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe("LandingPage With Enzyme", () => {
	it("render LandingPage component without crash", () => {
		const wrapper = shallow(<LandingPage/>);
		expect(wrapper.find("h1").text()).toEqual("OIENIV");
	});
});

describe("LandingPage With Snapshot", () => {
	it("render LandingPage component without crash", () => {
		const component = renderer.create(<LandingPage/>);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});

