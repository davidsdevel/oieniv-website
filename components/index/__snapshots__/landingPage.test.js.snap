// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage With Snapshot render LandingPage component without crash 1`] = `
<div
  className="jsx-3564708715"
  id="landing-page-container"
>
  <h1
    className="jsx-3564708715"
    id="title"
  >
    OIENIV
  </h1>
  <p
    className="jsx-3564708715"
    id="slogan"
  >
    Columna y Baluarte de la Verdad
  </p>
  <p
    className="jsx-3564708715"
    id="mini-history"
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </p>
</div>
`;
